src/v/debug_bundle/debug_bundle_service.cc

```diff
@@ -677,6 +713,26 @@ result<std::vector<ss::sstring>> service::build_rpk_arguments(
         rv.emplace_back(k8s_namespace_variable);
         rv.emplace_back(*params.k8s_namespace);
     }
+    if (
+      params.label_selector.has_value()
+      && !params.label_selector.value().empty()) {
+        rv.emplace_back(k8s_label_selector);
+        chunked_vector<ss::sstring> selectors;
```

Comment:
nit: It might be worth putting a reserve on here, although if you had a:
```c++
    friend std::ostream&
    operator<<(std::ostream& os, const label_selection& l) {
        fmt::print(os, "{}={}", l.key, l.value);
        return os;
    }
```
You could probably avoid this container.